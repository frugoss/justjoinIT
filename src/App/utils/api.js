export let API_HOST = 'http://localhost:7000'
if (process.env.NODE_ENV === 'test') {
  API_HOST = 'http://localhost'
}
