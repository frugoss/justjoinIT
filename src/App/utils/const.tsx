

export interface userInterface {
    auth?: boolean,
    name?: string,
    userID?: string,
    loggedPopup?: boolean,
    offPopup?: boolean,
    createPopup?: boolean,
    addPopup?: boolean
}


export interface offerInterface {
    _id: string,
    agreements: string,
    city: string,
    company: string,
    companyIndustry: string,
    companySize: string,
    coordinates: [{ lat: number, lng: number }],
    currency: string,
    description: string,
    email: string,
    employment: string,
    experience: string,
    logo: string | undefined,
    maxSalary: number,
    minSalary: number,
    street: string,
    techStack: [{ language: string, lvl: number }],
    technology: [{ color: string, background: string, img: string, name: string }],
    title: string,
    userID: string,
    website: string,
}

export interface formInterface {
    _id?: string,
    agreements?: string,
    city?: string,
    company?: string,
    companyIndustry?: string,
    companySize?: string,
    coordinates?: [{ lat: number, lng: number }],
    currency?: string,
    description?: string,
    email?: string,
    employment?: string,
    experience?: string,
    logo?: string| undefined,
    maxSalary?: number,
    minSalary?: number,
    street?: string,
    techStack: { language?: string, lvl?: number }[],
    technology?: [{ color: string, background: string, img: string, name: string }],
    title?: string,
    userID: string,
    website?: string,
}


export interface applicationInterface {
    checkbox: boolean,
    email: string,
    file: string,
    message: string,
    name: string,
    offerID: string
}

export interface filtersInterface{
    city: string,
    language: string,
    experience: string,
    salarymax: number,
    salarymin: number,
}

export interface matchInterface{
    city: string,
    language: string,
    experience: string,
    salarymax: string,
    salarymin: string,
}

export interface formValuesCV  {
    name:string,
    email:string,
    message: string,
    checkbox: string,
    file: File,
    offerID: string
}

export interface languageInterface {
    name: string,
    to: string,
    img: string,
    color: string,
    background: string
}



export interface mobileViewModeInterface {
    view: string,
    header: boolean
}

export interface cityInterface {
    name: string,
    to: string,
    coordinates: {lat:number, lng:number}
}
