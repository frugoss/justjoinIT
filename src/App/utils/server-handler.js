import {rest} from 'msw'
import {offersList, CV} from 'App/utils/mockData.js'

const handlers = [
  rest.post('/devs', async (req, res, ctx) => {
    return res(ctx.text("4325325325235"))
  }),
  rest.post('/devs/signup', async (req, res, ctx) => {
    return res(ctx.text("4325325325235"))
  }),
  rest.get('/', async (req, res, ctx) => {
    return res(ctx.json(offersList))
  }),
  rest.get('/cv/get', async (req, res, ctx) => {
    return res(ctx.json(CV))
  })
]

export {handlers}
