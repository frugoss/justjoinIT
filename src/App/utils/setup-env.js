import {server} from 'App/utils/server.js'
beforeAll(() => server.listen())
afterEach(() => server.resetHandlers())
afterAll(() => server.close())
