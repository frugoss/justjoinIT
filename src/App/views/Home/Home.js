import React, {useEffect, useState} from 'react';
import Filters from './Filters';
import {withRouter} from 'react-router-dom';
import styles from './home.module.scss'

const Home = ({location, match, history, offers, fetching}) => {
    const [filters, setFilters] = useState({ city: match.params.city || "all",
        language: match.params.language || "all",
        experience: match.params.experience || "all",
        salarymin: match.params.salarymin || 0,
        salarymax: match.params.salarymax || 50,
    });

useEffect(() => {
    if (location.pathname === "/"){
        setFilters({city: "all", language: "all", experience: "all", salarymin: 0, salarymax: 50})
    }
}, [location.pathname])


    const updateFilter = (key,value) => {
        setFilters({...filters, [key]:value})
    };

    return (
        <div className={styles.filters}>
            <Filters offers={offers} fetching={fetching} updateFilters={updateFilter} setFilters={setFilters} filters={filters} history={history} match={match}/>
        </div>


    )
};


export default withRouter(Home)
