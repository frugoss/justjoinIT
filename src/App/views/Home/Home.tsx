import React, {useEffect, useState} from 'react';
import Filters from './Filters';
import {withRouter, RouteComponentProps} from 'react-router-dom';
import styles from './home.module.scss'
import {matchInterface, offerInterface} from "../../utils/const";



interface HomeProps extends RouteComponentProps<matchInterface>{
    offers: offerInterface[],
    fetching: boolean
}


const Home: React.FC<HomeProps> = ({location, match, offers, fetching, history}) => {
    const [filters, setFilters] = useState({ city: match.params.city || "all",
        language: match.params.language || "all",
        experience: match.params.experience || "all",
        salarymin: parseInt(match.params.salarymin, 10) || 0,
        salarymax: parseInt(match.params.salarymax, 10) || 50,
    });

useEffect(() => {
    if (location.pathname === "/"){
        setFilters({city: "all", language: "all", experience: "all", salarymin: 0, salarymax: 50})
    }
}, [location.pathname])


    const updateFilter = (key:string,value:string | number) => {
        setFilters({...filters, [key]:value})
    };

    return (
        <div className={styles.filters}>
            <Filters history={history} offers={offers} fetching={fetching} updateFilters={updateFilter} setFilters={setFilters} filters={filters}/>
        </div>


    )
};


export default withRouter(Home)
