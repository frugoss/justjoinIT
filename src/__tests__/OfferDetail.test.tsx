import React from 'react';
import "@testing-library/jest-dom/extend-expect"
import  {OfferDetail} from "../App/views/Home/OfferDetail";
import {screen, render} from '@testing-library/react';
import {offersList} from '../App/utils/mockData'
import userEvent from "@testing-library/user-event";

test("Sending a CV", async () => {
    const match = {params:{offerTitle:"lion-python-developer"}, isExact:true, path:"/offers/:offerTitle",url:"/offers/lion-python-developer"}
    render(<OfferDetail offersList={offersList} match={match} />)
    userEvent.type(screen.getByLabelText("First and last name *"),"Andrzej Tunaj")
    userEvent.type(screen.getByLabelText("Email *"),"mail@gmail.com")
    userEvent.click(screen.getByRole("button", {name: /apply cv/i}))
    await expect(screen.findByText("Great! Your application was successfully sent to: Lion"))
})

test("Snapshot",   () => {
    const match = {params:{offerTitle:"lion-python-developer"}, isExact:true, path:"/offers/:offerTitle",url:"/offers/lion-python-developer"}
    const {container} = render(<OfferDetail offersList={offersList} match={match} />);
    expect(container).toMatchSnapshot();
})

