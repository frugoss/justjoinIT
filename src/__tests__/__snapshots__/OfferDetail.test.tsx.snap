// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot 1`] = `
<div>
  <div
    class="offerDetailContainer"
  >
    
    <div
      class="offer mobilePadding"
    >
      <div
        class="top"
      >
        <div
          class="banner"
        >
          <div
            class="backButton"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-arrow-16"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"
              />
            </svg>
          </div>
          <div
            class="name"
          >
            <div
              class="logoCircle"
            >
              <div
                class="circle"
              >
                <img
                  alt="logo"
                  class="companyLogoDimensions"
                  src="data:image/png;"
                />
              </div>
            </div>
            <div
              class="titleSalary"
            >
              <span
                class="salaryStreetDetail"
              >
                <span>
                  3 000
                </span>
                 - 
                <span>
                  10 000
                </span>
                 
                EUR
                 net/month
              </span>
              <div
                class="title"
              >
                <span
                  class="titleDetail"
                >
                  Python Developer
                </span>
              </div>
              <div>
                <span
                  class="salaryStreetDetail"
                >
                  Tuwima
                  , 
                  Gda≈Ñsk
                   
                </span>
              </div>
            </div>
            <div
              class="flexJustify"
            >
              <div
                class="apply"
              >
                <a
                  class="linkRoute"
                  href="#apply"
                >
                  <button
                    class="MuiButtonBase-root"
                    tabindex="0"
                    type="button"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 4l-8 5-8-5V6l8 5 8-5v2z"
                      />
                    </svg>
                     Apply 
                  </button>
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="itSkills"
        >
          <div
            class="mobileColumn"
          >
            <div
              class="skill"
            >
              <div
                class="skillCircle"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-businessIcon-17"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 7V3H2v18h20V7H12zM6 19H4v-2h2v2zm0-4H4v-2h2v2zm0-4H4V9h2v2zm0-4H4V5h2v2zm4 12H8v-2h2v2zm0-4H8v-2h2v2zm0-4H8V9h2v2zm0-4H8V5h2v2zm10 12h-8v-2h2v-2h-2v-2h2v-2h-2V9h8v10zm-2-8h-2v2h2v-2zm0 4h-2v2h2v-2z"
                  />
                </svg>
              </div>
              <a
                class="linkRoute"
                href="https://www.lion.com"
                rel="noopener noreferrer"
                target="_blank"
              >
                Lion
              </a>
              <div
                class="companyDetails"
              >
                Company Name
              </div>
            </div>
            <div
              class="skill"
            >
              <div
                class="skillCircle"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-peopleIcon-18"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16 11c1.66 0 2.99-1.34 2.99-3S17.66 5 16 5c-1.66 0-3 1.34-3 3s1.34 3 3 3zm-8 0c1.66 0 2.99-1.34 2.99-3S9.66 5 8 5C6.34 5 5 6.34 5 8s1.34 3 3 3zm0 2c-2.33 0-7 1.17-7 3.5V19h14v-2.5c0-2.33-4.67-3.5-7-3.5zm8 0c-.29 0-.62.02-.97.05 1.16.84 1.97 1.97 1.97 3.45V19h6v-2.5c0-2.33-4.67-3.5-7-3.5z"
                  />
                </svg>
              </div>
              <span>
                10
              </span>
              <div
                class="companyDetails"
              >
                Company Size
              </div>
            </div>
          </div>
          <div
            class="mobileColumn"
          >
            <div
              class="skill"
            >
              <div
                class="skillCircle"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-fileIcon-19"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M6 2c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H18c1.1 0 2-.9 2-2V8l-6-6H6zm7 7V3.5L18.5 9H13z"
                  />
                </svg>
              </div>
              <span>
                Permanent
              </span>
              <div
                class="companyDetails"
              >
                EMP. type
              </div>
            </div>
            <div
              class="skill"
            >
              <div
                class="skillCircle"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-chartIcon-20"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M3.5 18.49l6-6.01 4 4L22 6.92l-1.41-1.41-7.09 7.97-4-4L2 16.99z"
                  />
                </svg>
              </div>
              <div>
                Mid
              </div>
              <div
                class="companyDetails"
              >
                EXP. lvl
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="offerDetails"
      >
        <div
          class="techStack"
        >
          <div
            class="titleTech"
          >
            Tech Stack
          </div>
          <div
            class="insideTech"
          >
            <div
              class="techSkills"
            >
              <div
                class="techPadding"
              >
                <div
                  class="flexRow"
                >
                  <span
                    class="dotClicked"
                  />
                  <span
                    class="dotClicked"
                  />
                  <span
                    class="dotClicked"
                  />
                  <span
                    class="dot"
                  />
                  <span
                    class="dot"
                  />
                </div>
                <div
                  class="language"
                >
                  Javascript
                </div>
                <div
                  class="desc"
                >
                  Regular
                </div>
              </div>
              <div
                class="techPadding"
              >
                <div
                  class="flexRow"
                >
                  <span
                    class="dotClicked"
                  />
                  <span
                    class="dotClicked"
                  />
                  <span
                    class="dot"
                  />
                  <span
                    class="dot"
                  />
                  <span
                    class="dot"
                  />
                </div>
                <div
                  class="language"
                >
                  C#
                </div>
                <div
                  class="desc"
                >
                  Junior
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="sectionCard"
        >
          <div
            class="titleTech"
          >
            Description
          </div>
          <div
            class="description"
          >
            <div
              class="additionalDesc"
            >
              <div>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit. In tincidunt maximus sem vitae cursus. Integer ullamcorper quam quis molestie dictum. Nam eros augue, aliquam vitae tristique in, laoreet ac massa. Suspendisse ultricies vulputate elit, scelerisque pellentesque turpis tempor in. Donec neque est, feugiat sit amet tempor ut, gravida sit amet ligula. Donec elit ante, pharetra ac ex a, malesuada molestie velit. Vivamus cursus risus mi, quis maximus justo convallis vitae.  Nullam ultrices porttitor quam, non cursus risus convallis vel. Mauris feugiat lectus eu arcu tempor, eget hendrerit eros imperdiet. Ut vitae sodales nisi. Nam eu dolor eu libero tincidunt maximus non vitae odio. Cras neque orci, dignissim eu feugiat ac, consectetur ut nisl. Maecenas euismod sem orci. Phasellus eget nulla et sapien vulputate mollis vel tempor sem. Nulla cursus pharetra lacus, lobortis porta nulla eleifend et. Curabitur quis arcu id velit gravida euismod vitae eu turpis. In suscipit, lectus vitae maximus iaculis, mauris massa porttitor est, vel porttitor orci massa ut quam. Morbi in sapien sit amet nisl faucibus posuere.  Quisque sit amet finibus leo. Aenean placerat sit amet nulla rutrum auctor. Curabitur scelerisque eleifend purus, quis bibendum lectus venenatis vitae. Quisque pellentesque lorem ex, eu facilisis magna convallis eget. Aliquam eget fermentum augue. Ut sollicitudin dui elit, eu consectetur mi rutrum in. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Nam at massa interdum, feugiat augue ac, fringilla mauris. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus ut velit vulputate nulla luctus aliquet.  Sed ornare viverra felis vitae tincidunt. Pellentesque hendrerit massa sit amet nibh viverra lacinia. Duis vitae metus vel libero tristique euismod. Sed congue consequat purus, quis convallis ex venenatis id. Donec lacus massa, tincidunt vel erat quis, accumsan euismod erat. Curabitur leo dolor, aliquet eget fringilla a, fermentum pulvinar nisl. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.  Proin eleifend sodales ligula in hendrerit. Nam volutpat, enim id varius bibendum, metus orci venenatis nibh, nec vestibulum ex dolor sit amet ipsum. Cras blandit enim vitae justo lobortis, vel efficitur sem luctus. Duis non dolor id erat consectetur eleifend. Nullam ultrices dictum lacus faucibus aliquet. Vestibulum eget diam cursus, fermentum turpis id, aliquet odio. Quisque vitae bibendum nibh, faucibus malesuada mi. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Mauris dictum risus vel metus convallis, vitae cursus metus faucibus. Cras placerat rhoncus enim non tempor. Integer augue odio, efficitur non diam et, luctus lobortis mi. Suspendisse laoreet vestibulum ex non pretium. Nullam rhoncus, risus nec placerat laoreet, dui neque ultrices metus, vitae congue ligula quam a magna. Nullam sodales lacus in aliquet sagittis.
              </div>
            </div>
          </div>
        </div>
        <div
          class="sectionCard"
        >
          <div
            class="titleTech"
          >
            Apply for this job
          </div>
          <form
            novalidate=""
          >
            <div
              class="cvContainer"
              id="apply"
            >
              <div
                class="inputsDirection"
              >
                <div
                  class="inputs marginName"
                >
                  <div
                    class="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
                  >
                    <label
                      class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined Mui-required Mui-required"
                      data-shrink="true"
                      for="name"
                      id="name-label"
                    >
                      First and last name
                      <span
                        aria-hidden="true"
                        class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
                      >
                        ‚Äâ
                        *
                      </span>
                    </label>
                    <div
                      class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedStart MuiOutlinedInput-adornedStart"
                    >
                      <div
                        class="MuiInputAdornment-root MuiInputAdornment-positionStart"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M12 5.9c1.16 0 2.1.94 2.1 2.1s-.94 2.1-2.1 2.1S9.9 9.16 9.9 8s.94-2.1 2.1-2.1m0 9c2.97 0 6.1 1.46 6.1 2.1v1.1H5.9V17c0-.64 3.13-2.1 6.1-2.1M12 4C9.79 4 8 5.79 8 8s1.79 4 4 4 4-1.79 4-4-1.79-4-4-4zm0 9c-2.67 0-8 1.34-8 4v3h16v-3c0-2.66-5.33-4-8-4z"
                          />
                        </svg>
                      </div>
                      <input
                        aria-invalid="false"
                        class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedStart MuiOutlinedInput-inputAdornedStart"
                        id="name"
                        name="name"
                        required=""
                        type="text"
                        value=""
                      />
                      <fieldset
                        aria-hidden="true"
                        class="PrivateNotchedOutline-root-21 MuiOutlinedInput-notchedOutline"
                      >
                        <legend
                          class="PrivateNotchedOutline-legendLabelled-23 PrivateNotchedOutline-legendNotched-24"
                        >
                          <span>
                            First and last name
                            ¬†*
                          </span>
                        </legend>
                      </fieldset>
                    </div>
                  </div>
                </div>
                <div
                  class="inputs marginMail"
                >
                  <div
                    class="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
                  >
                    <label
                      class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined Mui-required Mui-required"
                      data-shrink="true"
                      for="email"
                      id="email-label"
                    >
                      Email
                      <span
                        aria-hidden="true"
                        class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
                      >
                        ‚Äâ
                        *
                      </span>
                    </label>
                    <div
                      class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedStart MuiOutlinedInput-adornedStart"
                    >
                      <div
                        class="MuiInputAdornment-root MuiInputAdornment-positionStart"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 4l-8 5-8-5V6l8 5 8-5v2z"
                          />
                        </svg>
                      </div>
                      <input
                        aria-invalid="false"
                        class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedStart MuiOutlinedInput-inputAdornedStart"
                        id="email"
                        name="email"
                        required=""
                        type="text"
                        value=""
                      />
                      <fieldset
                        aria-hidden="true"
                        class="PrivateNotchedOutline-root-21 MuiOutlinedInput-notchedOutline"
                      >
                        <legend
                          class="PrivateNotchedOutline-legendLabelled-23 PrivateNotchedOutline-legendNotched-24"
                        >
                          <span>
                            Email
                            ¬†*
                          </span>
                        </legend>
                      </fieldset>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="introCV"
              >
                <div
                  class="about"
                >
                  <div
                    class="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
                  >
                    <label
                      class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined"
                      data-shrink="true"
                    >
                      Introduce yourself (linkedin/github links)
                    </label>
                    <div
                      class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-multiline MuiOutlinedInput-multiline MuiInputBase-adornedStart MuiOutlinedInput-adornedStart"
                    >
                      <div
                        class="MuiInputAdornment-root MuiInputAdornment-positionStart"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
                          />
                        </svg>
                      </div>
                      <textarea
                        aria-invalid="false"
                        class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMultiline MuiOutlinedInput-inputMultiline MuiInputBase-inputAdornedStart MuiOutlinedInput-inputAdornedStart"
                        name="message"
                        rows="4"
                      />
                      <fieldset
                        aria-hidden="true"
                        class="PrivateNotchedOutline-root-21 MuiOutlinedInput-notchedOutline"
                      >
                        <legend
                          class="PrivateNotchedOutline-legendLabelled-23 PrivateNotchedOutline-legendNotched-24"
                        >
                          <span>
                            Introduce yourself (linkedin/github links)
                          </span>
                        </legend>
                      </fieldset>
                    </div>
                  </div>
                </div>
                <div
                  class="CV"
                >
                  <section
                    class="container"
                  >
                    <div
                      class="dropzone"
                      tabindex="0"
                    >
                      <input
                        accept="application/pdf"
                        autocomplete="off"
                        multiple=""
                        style="display: none;"
                        tabindex="-1"
                        type="file"
                      />
                      <svg
                        aria-hidden="true"
                        class="svg-inline--fa fa-file-upload fa-w-12 fa-3x pictureDrop"
                        data-icon="file-upload"
                        data-prefix="fas"
                        focusable="false"
                        role="img"
                        viewBox="0 0 384 512"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M224 136V0H24C10.7 0 0 10.7 0 24v464c0 13.3 10.7 24 24 24h336c13.3 0 24-10.7 24-24V160H248c-13.2 0-24-10.8-24-24zm65.18 216.01H224v80c0 8.84-7.16 16-16 16h-32c-8.84 0-16-7.16-16-16v-80H94.82c-14.28 0-21.41-17.29-11.27-27.36l96.42-95.7c6.65-6.61 17.39-6.61 24.04 0l96.42 95.7c10.15 10.07 3.03 27.36-11.25 27.36zM377 105L279.1 7c-4.5-4.5-10.6-7-17-7H256v128h128v-6.1c0-6.3-2.5-12.4-7-16.9z"
                          fill="currentColor"
                        />
                      </svg>
                      <h2
                        class="uploadDrop"
                      >
                        Upload CV (.pdf)
                      </h2>
                    </div>
                    <aside />
                  </section>
                </div>
              </div>
              <div
                class="checkBox"
              >
                <div
                  class="MuiFormGroup-root"
                >
                  <div
                    class="MuiFormControl-root"
                  >
                    <div
                      class="MuiFormGroup-root"
                    >
                      <label
                        class="MuiFormControlLabel-root"
                      >
                        <span
                          aria-disabled="false"
                          class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-25 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                        >
                          <span
                            class="MuiIconButton-label"
                          >
                            <input
                              class="PrivateSwitchBase-input-28"
                              data-indeterminate="false"
                              name="checkbox"
                              type="checkbox"
                              value=""
                            />
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                              />
                            </svg>
                          </span>
                        </span>
                        <span
                          class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                        >
                          Processing data in future recruitment
                        </span>
                      </label>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="bottomSection"
              >
                <p
                  class="captchaStyling"
                >
                  This site is protected by reCAPTCHA and the Google Privacy Policy and Terms of Service apply.
                </p>
                <button
                  class="MuiButtonBase-root makeStyles-root-15"
                  color="primary"
                  tabindex="0"
                  type="submit"
                >
                  Apply CV
                </button>
              </div>
            </div>
          </form>
        </div>
      </div>
    </div>
  </div>
</div>
`;
